test/utils.js

```diff
@@ -126,6 +163,7 @@ describe('FlowLimit', async () => {
         const latest = Number(await time.latest());
         const epoch = 6 * 3600;
         const next = (Math.floor(latest / epoch) + 1) * epoch;
+
         await time.increaseTo(next);
     }
```

Comment:
The two tests that use this helper function fail on live testnets due to the `increaseTo` function. Any ideas for how to modify this function to accommodate live testnets?

Reply:
hmm, maybe we need to inherit a test token contract that overrides the flow limit to a smaller value